### PR TITLE
Update UG for info, calculateCAP, calculateMC

### PR DESCRIPTION
**Pull request by ji-just-ji**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -315,16 +314,26 @@ The **module code, module title, modular credit and module description** of the
 
 Format: `info MODULE`
 
-Examples:
-* `info CS2109S`
-* `info CS1010`
+<div class="alert alert-warning"><md>:gear: Example:
+* info CS2109S
+* info CS1010
```
Comment (marquestye) [inline docs/UserGuide.md:319]: Should include explanation for each example command.

File: docs/diagrams/AddModuleActivityDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+:Parse commandString;
+
+if () then ([else])
+    :ParseException: Invalid Command Format;
+    stop
+( [all fields are filled and valid]  ) elseif() then ([else]))
```
Comment (marquestye) [inline docs/diagrams/AddModuleActivityDiagram.puml:17]: Extra `)`

File: docs/diagrams/AddModuleActivityDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+:User executes add command;
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+:Parse commandString;
+
+if () then ([else])
+    :ParseException: Invalid Command Format;
+    stop
```
Comment (marquestye) [inline docs/diagrams/AddModuleActivityDiagram.puml:16]: Can try adding puml partitions (AddCommandParser and AddCommand) to show where everything happens.

File: docs/diagrams/AddModuleActivityDiagram.puml
Comment (marquestye) [inline docs/diagrams/AddModuleActivityDiagram.puml:1]: image

Theres an extra diamond at the bottom for me.

**Review by xCOLOURx — APPROVED** [review]
LGTM 👍

**Pull request merged**